server/src/main/java/io/seata/server/coordinator/DefaultCore.java

```diff
@@ -182,9 +180,7 @@ public GlobalStatus commit(String xid) throws TransactionException {
     public boolean doGlobalCommit(GlobalSession globalSession, boolean retrying) throws TransactionException {
         boolean success = true;
         // start committing event
-        eventBus.post(new GlobalTransactionEvent(globalSession.getTransactionId(), GlobalTransactionEvent.ROLE_TC,
-            globalSession.getTransactionName(), globalSession.getApplicationId(), globalSession.getTransactionServiceGroup(),
-            globalSession.getBeginTime(), null, globalSession.getStatus()));
+        SessionHelper.postTcSessionBeginEvent(globalSession);
```

Comment:
`postTcSessionBeginEvent`  is not appropriate here.

Reply:
optimize the method naming.

Reply:
will optimize later